src/Surfnet/StepupSelfService/SelfServiceBundle/Value/GsspToken.php

```diff
@@ -63,7 +63,7 @@ private function __construct(ViewConfig $viewConfig, $type)
      */
     public function getRoute()
     {
-        return 'ss_registration_gssf_initiate';
+        return 'ss_registration_gssf_authenticate';
```

Comment:
The heart of this change lies here. The route is updated to directly reference the `ss_registration_gssf_authenticate` action. Skipping the initiate route.